pkg/cmd/admin/node/evacuate.go

```diff
@@ -0,0 +1,134 @@
+package node
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/fields"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	kerrors "github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
+)
+
+type EvacuateOptions struct {
+	Options *NodeOptions
+
+	// Optional params
+	DryRun bool
+	Force  bool
+}
+
+func (e *EvacuateOptions) AddFlags(cmd *cobra.Command) {
+	flags := cmd.Flags()
+
+	flags.BoolVar(&e.DryRun, "dry-run", false, "Show pods that will be migrated. Optional param for --evacuate")
+	flags.BoolVar(&e.Force, "force", false, "Delete pods not backed by replication controller. Optional param for --evacuate")
+}
+
+func (e *EvacuateOptions) Run() error {
+	nodes, err := e.Options.GetNodes()
+	if err != nil {
+		return err
+	}
+
+	errList := []error{}
+	for _, node := range nodes {
+		err := e.RunEvacuate(node)
+		if err != nil {
+			// Don't bail out if one node fails
+			errList = append(errList, err)
+		}
+	}
+	if len(errList) != 0 {
```

Comment:
followup: you can just `return kerrors.NewAggregate(errList)` with no checking.  empty slices return nil.